blocks/subscriptions-block/components/PaymentForm/index.story.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import { Elements } from '@stripe/react-stripe-js';
+import { loadStripe } from '@stripe/stripe-js';
+
+import PaymentForm from '.';
+
+export default {
+  title: 'Blocks/Subscriptions/Components/Payment Form',
+  parameters: {
+    chromatic: { viewports: [320, 1200] },
+  },
+};
+
+// test string via stripe
+const stripePromise = loadStripe('pk_test_TYooMQauvdEDq54NiTphI7jx');
```

Comment:
wrapped the stripe promise and elements around the stripe form to ensure it's renderable

Reply:
couldn't find if we had other status sections for forms

Reply:
Not sure if something like this would be helpful: 

```
        {formErrorText ? (
          <section className="xpmedia-form-error" role="alert">
            <PrimaryFont as="p">
              <ErrorIcon />
              {formErrorText}
            </PrimaryFont>
          </section>
        ) : null}
```

Reply:
did you mean to put this comment in the story code?

Reply:
oh, no. this might be a good addition to the actual form ... `Not sure if something like this would be helpful:`

Reply:
are you asking about the `// test string via stripe`? I can write more on this but needed a fake stripe string here for testing with storybook